### PR TITLE
#171380739 setup dotenv

### PR DESCRIPTION
**Pull request by since9629**
#### What does this PR do?
Setup dotenv as a way to load environment variables from .env file into process.env
#### Description of Task to be completed?
- add more environment variables to .env.example
- Install `dotenv` npm package to load .env file KEY=VALUE content into process.env
#### How should this be manually tested?
- After cloning the project repo, run `npm i` inside the cloned repo folder
- run the following commands respectively:
`git fetch`
`git checkout ch-setup-dotenv-171380739`
`npm run start`
- If the above steps have not been followed correctly, the app should throw an error
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#171380739
#### Screenshots (if appropriate)
N/A

### CODE REVIEW

**Review by Muhire-Josue — COMMENTED** [review]
Why did you include passport? I can see your PR is to setup dotenv. great job by the way

**Comment by since9629** [comment]
> Why did you include passport? I can see your PR is to setup dotenv. great job by the way

@Muhire-Josue the passport.js file was included by default,check the develop branch

**Review by Muhire-Josue — APPROVED** [review]
LGTM

**Comment by kipropbrian** [comment]
Your PR contains more than .env.example . Please make sure that your PR contains only relevant code to your PR

**Comment by since9629** [comment]
> Your PR contains more than .env.example . Please make sure that your PR contains only relevant code to your PR

Could you elaborate on *Your PR contains more than .env.example*

**Review by the22mastermind — APPROVED** [review]
Looks good to me. Well done @karamuka 
Keep up the good work.

File: .env.example [hunk 1]
```diff
@@ -1,6 +1,25 @@
+APP_NAME=Barefoot nomad
+NODE_ENV=# development | production | test
+PORT=
+SECRET=
+
```
Comment (ricahcyuzuzo) [inline .env.example]: @karamuka Would it be good if you put these codes below the ones for development?

File: package.json [hunk 2]
```diff
@@ -21,11 +22,10 @@
     "chai-http": "^4.3.0",
```
Comment (descholar-ceo) [inline package.json:23]: Could you move this `chai` should be to devDependecies?

File: package.json [hunk 3]
```diff
@@ -47,4 +47,4 @@
     "request": "^2.88.2",
     "underscore": "^1.9.2"
   }
-}
+}
```
Comment (ricahcyuzuzo) [inline package.json]: Add a newline!

**Review by ricahcyuzuzo — APPROVED** [review]

**Review by Ugizwenayo-Divine — APPROVED** [review]

**Review by muchristian — APPROVED** [review]

**Review by descholar-ceo — APPROVED** [review]
LGTM

File: package.json (hunk 3 above)
Comment (mifeille) [inline package.json]: Please add a new line here

**Pull request merged**